libass/ass_compat.h

```diff
@@ -26,4 +26,15 @@
 #define inline __inline
 #endif
 
+#ifndef HAVE_STRDUP
+char *ass_strdup_fallback(const char *s); //definition in ass_utils.c
```

Comment:
Added spaces, thx.
If with “inplace inline” you meant `static inline` function definitions inside `ass_compat.h`, I decided against that, because it results in identical two functions per source file being placed in the resulting lib *(if fallbacks are required)*. Also the "proper" `str(n)dup` functions aren't forced inline either; leaving this to the compiler and possibly LTO seems better overall imo.